app/services/user_validation_service.rb

```diff
@@ -87,7 +87,7 @@ def missing_user_features(db_user)
   end
 
   def user_is_super_admin?
```

Comment:
a) This calls into that helper. Since that is not visible, it uses `controller#super_admin_user` to access it.
b) I have the goal of setting `user#current_group` correctly, then we can just use `db_user#super_admin_user?` and dump the helper / call into the controller.